modules/Bio/EnsEMBL/Production/Pipeline/AlphaFold/HiveLoadAlphaFoldDBProteinFeatures.pm

```diff
@@ -110,6 +111,10 @@ sub fetch_input {
 
   $self->hrdb_set_con($core_dba,"core");
 
+  print "Cleaning up old protein features and analysis\n" if defined($self->param('debug'));
```

Comment:
is print going to log in Hive log table?

Reply:
Not sure: that's why the IF with the "debug" parameter, which is by default undefined.
I used it on STDOUT with `standaloneJob.pl`.
I wanted to double-check with Arne when he's back next week.

If you agree, I can fix it for 109. If you don't, I'll remove the "print" altogether.

Reply:
I believe this can be change to log instead:
```use Bio::EnsEMBL::Utils::Exception qw(throw info);```
Then use 
```info(sprintf("No data found for species %s in GIFTS DB using endpoint %s and assembly %s. Message:\n%s", $self->{'species'}, $self->{'rest_server'}, $self->{'cs_version'}, $@));```

Issued from past Arne PR :-)

Reply:
If too complicated, just comment the lines, and we'll make the change another time.

Reply:
Just pushed changes. Tested OK for "candida albicans" only for the sake of being quick.